contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2270,6 +2271,10 @@ static void bbf_ProcessUtility(PlannedStmt *pstmt,
 							}
 							else if (strcmp(defel->defname, "from_windows") == 0)
 							{
+								if (!pltsql_is_windows_allowed)
```

Comment:
For safe side, we should also have a similar check for user/role at line no 2405

Reply:
Done!